features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/notificationsettings/RoomNotificationSettingsPresenter.kt

```diff
@@ -87,6 +90,7 @@ class RoomNotificationSettingsPresenter @AssistedInject constructor(
             getDefaultRoomNotificationMode(defaultRoomNotificationMode)
             fetchNotificationSettings(pendingRoomNotificationMode, roomNotificationSettings)
             observeNotificationSettings(pendingRoomNotificationMode, roomNotificationSettings)
+            shouldDisplayMentionsOnlyDisclaimer = !notificationSettingsService.canHomeServerPushEncryptedEventsToDevice().getOrDefault(true)
```

Comment:
https://github.com/element-hq/element-x-ios/issues/2078 requests that the disclaimer should be shown only for encrypted room, so I think a test is missing here.

Could be changed to:

```suggestion
            shouldDisplayMentionsOnlyDisclaimer = room.isEncrypted && !notificationSettingsService.canHomeServerPushEncryptedEventsToDevice().getOrDefault(true)
```